Automattic/wp-calypso
# Subscription settings: Turn off the emails check

## Proposed Changes

This PR adds the checkbox to turn off the reception of emails to which the user is subscribed. This is the result:

<img width="705" alt="Screenshot 2023-03-15 at 19 38 24" src="https://user-images.githubusercontent.com/3832570/225409794-4e9e97ec-c19c-45df-93ce-281efb8bc66d.png">

## Testing Instructions

1. Apply this PR and start the application (`yarn start`).
2. Go to `http://calypso.localhost:3000/subscriptions/settings`.
3. You should see the Block Emails setting there:
<img width="627" alt="Screenshot 2023-03-16 at 12 03 35" src="https://user-images.githubusercontent.com/3832570/225598187-4f94f3d0-dd32-4798-a1c2-ecdb9865e848.png">

So that you know, there is no interaction in the option at the moment. When you click on the checkbox, nothing happens. That interactivity will be implemented in other PR.

Thanks for the review! ❤️

Fixes https://github.com/Automattic/wp-calypso/issues/74471
